packages/cli/src/ui/App.tsx

```diff
@@ -420,15 +421,16 @@ const App = ({ config, settings, startupWarnings = [], version }: AppProps) => {
 
   // Terminal and UI setup
   const { rows: terminalHeight, columns: terminalWidth } = useTerminalSize();
+  const isNarrow = terminalWidth < NARROW_BREAKPOINT;
   const { stdin, setRawMode } = useStdin();
   const isInitialMount = useRef(true);
 
   const widthFraction = 0.9;
   const inputWidth = Math.max(
     20,
-    Math.floor(terminalWidth * widthFraction) - 3,
+    Math.floor(terminalWidth * widthFraction) - 4,
   );
-  const suggestionsWidth = Math.max(60, Math.floor(terminalWidth * 0.8));
+  const suggestionsWidth = Math.max(20, Math.floor(terminalWidth * 0.9));
```

Comment:
now that I think about it, we likely never want this to exceed the terminalWidth as that won't render well. Probably really just want
`Math.min(Math.max(20, terminalWidth * 0.8), terminalWidth)` or something
@miguelsolorio might have better ideas. it may be silly that we ever made this smaller than inputWidth.